### PR TITLE
style fixes & pixel perfect

### PR DESCRIPTION
**Pull request by nadavbh-moveo**
@sivanbasa93 @kennethmoveo

### CODE REVIEW

File: src/components/Card.tsx [hunk 1]
```diff
@@ -27,12 +27,16 @@ export default function Card({ cardData, type }: CardProps) {
                 ))}
               </div>
             </div>
-            <div className="price">&#8362;{cardData.price}</div>
```
Comment (sivanbasa93) [inline src/components/Card.tsx:30]: maybe is there prettier way to put space ?

File: src/components/Card.tsx [hunk 2]
```diff
@@ -27,12 +27,16 @@ export default function Card({ cardData, type }: CardProps) {
...
           </>
         ) : (
           ""
         )}
       </div>
+     { type === 'dish'? <div className="price">
+      <img src={lineSrc} alt="" />
+      &#8362;{cardData.price}
```
Comment (sivanbasa93) [inline src/components/Card.tsx:37]: same here

File: src/pages/HomePage.tsx [hunk 3]
```diff
@@ -12,7 +12,7 @@ export default function HomePage() {
   return (
     <div>
         <Hero />
-        <CardList cardListHeader={popularRestaurantHeader} cardListData={restaurantData} type='restaurant'/>
+        <CardList cardListHeader={popularRestaurantHeader} cardListData={restaurantData.slice(0, 3)} type='restaurant'/>
```
Comment (sivanbasa93) [inline src/pages/HomePage.tsx:15]: save restaurantData.slice(0, 3) in new variable

File: src/styles/basic/_layout.scss [hunk 4]
```diff
@@ -7,7 +7,7 @@
     }
 }
 
-.max-section-size {
-    max-width: 1086.5px;
-}
+// .max-section-size {
```
Comment (sivanbasa93) [inline src/styles/basic/_layout.scss:10]: delete comment

File: src/components/Card.tsx (hunk 1 above)
Comment (nadavbh-moveo) [reply src/components/Card.tsx:30]: done

File: src/components/Card.tsx (hunk 2 above)
Comment (nadavbh-moveo) [reply src/components/Card.tsx]: done

File: src/pages/HomePage.tsx (hunk 3 above)
Comment (nadavbh-moveo) [reply src/pages/HomePage.tsx]: done

File: src/styles/basic/_layout.scss (hunk 4 above)
Comment (nadavbh-moveo) [reply src/styles/basic/_layout.scss]: done

**Review by sivanbasa93 — APPROVED** [review]

**Pull request merged**